issue35/compare sun_abraham vs EventStudyInteract.do

```diff
@@ -0,0 +1,71 @@
+* compare sun_abraham.ado vs EventStudyInteract

```

Comment:
@Constantino-Carreto-Romero This file should not be merged to master. In fact, none of the files in the issue folder should be merged to master.

Reply:
In https://github.com/JMSLab/xtevent/commit/bf8eccbe65c5292900df6fe37b18ab0709528096, I deleted the issue folder